app/controllers/topical_events_controller.rb

```diff
@@ -4,9 +4,9 @@ class TopicalEventsController < ClassificationsController
   def show
     @classification = TopicalEvent.friendly.find(params[:id])
     @content_item = Whitehall.content_store.content_item(@classification.base_path)
-    @publications = fetch_associated(:published_publications, PublicationesquePresenter)
-    @consultations = fetch_associated(:published_consultations, PublicationesquePresenter)
-    @announcements = fetch_associated(:published_announcements, AnnouncementPresenter)
+    @publications = fetch_related_documents(filter_format: 'publication')
+    @consultations = fetch_related_documents(filter_format: 'consultation')
+    @announcements = fetch_related_documents(filter_format: 'news_article')
     @detailed_guides = @classification.published_detailed_guides.includes(:translations, :document).limit(5)
```

Comment:
In our work to switch over the announcements finder to use Rummager we've decided to filter for announcements by `content_store_document_type` as according to the documentation `format` may be deprecated, and it allows us to be more specific with which documents are returned from Rummager.  We've got them listed out here: https://github.com/alphagov/whitehall/pull/4449/commits/dcdbd71bc2702945a4e8aab21aefd9f89b7511c7
For consistency I think it would be good to filter in the same way here so that the 3 announcements that a user sees on this page are the top 3 announcements on the announcements finder page. What do you think?

Reply:
That's a good idea. After or as part of #4449) we can add the following to the topical events controller.

```
  def fetch_announcements
    # This matches behaviour at https://github.com/alphagov/whitehall/commit/dcdbd71bc2702945a4e8aab21aefd9f89b7511c7
    announcement_doctypes = Whitehall::AnnouncementFilterOption.all.map(&:document_type).flatten
    fetch_related_documents(filter_content_store_document_type: announcement_doctypes)
  end
```